installers/WindowsInstaller.hs

```diff
@@ -136,6 +139,8 @@ writeInstallerNSIS fullVersion = do
           , "DetailPrint \"liteFirewall::AddRule: $0\""
           ]
 
+        exec "build-certificates-win64.bat >build-certificates.log 2>&1"
```

Comment:
macOS installs certficates at installation time, while this line generates them at installer generation time.

Shouldn't it happen inside `daedalus.bat`, but only one time?